ui/src/main/webapp/src/app/misc/about.component.html

```diff
@@ -49,14 +49,14 @@ <h2 id="version">Version</h2>
                                 The <a target="_blank" href="https://access.redhat.com/documentation/en/red-hat-application-migration-toolkit/">documentation</a>
                                 contains information and guides to help you learn about Red Hat Application Migration Toolkit
```

Comment:
Remove ` and guides ` part (see spreadsheet line 36)

https://docs.google.com/spreadsheets/d/1Xo1v_VpZmCfLIx5ySX-gyBK1w0uPTdntHofqRNU5fNE/edit#gid=0